### PR TITLE
change interface to support command line args

### PR DESCRIPTION
**Pull request by zancas**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,43 +1,30 @@
...
+    let raw_command_help = std::str::from_utf8(&command_help_output.stdout)
+        .expect("Invalid raw_command_help, error!");
 
-        log_raw_output(&command, raw_command_help.to_string());
-        //select just for blessed results.
-        if blessed_tome.contains(&command) {
-            //dbg!(&command);
-            produce_interpretation(raw_command_help);
+    log_raw_output(command, raw_command_help.to_string());
+    //select just for blessed results.
```
Comment (dannasessha) [inline src/main.rs:14]: `blessed` is not used in main anymore.

File: src/main.rs [hunk 2]
```diff
@@ -1,43 +1,30 @@
...
+    produce_interpretation(raw_command_help);
+    dbg!(command);
+}
+fn main() {
+    let args = &std::env::args().collect::<Vec<String>>()[1..];
```
Comment (dannasessha) [inline src/main.rs:19]: consider adding a comment here for 'proper usage' in the new quizface development scheme.

File: src/lib.rs [hunk 3]
```diff
@@ -62,6 +62,7 @@ fn record_interpretation(cmd_name: String, interpretation: serde_json::Value) {
         utils::logging::create_version_name(),
         cmd_name
     );
+    dbg!(&interpretation);
```
Comment (dannasessha) [inline src/lib.rs:65]: Remove this dbg?

Comment (zancas) [reply src/lib.rs:65]: Received permission.

**Pull request merged**